src/libs/ck-libs/io/fs_parameters.c

```diff
@@ -43,7 +43,8 @@ size_t CkGetFileStripeSize(const char *filename) {
   }
 
   if (rc != 0) {
-    CkAbort("[CkIO] llapi_file_get_stripe error");
+    CkPrintf("[CkIO] Cannot extract lustre file stripe size, using default of 4 MB\n");
```

Comment:
Is this function called on every PE, or only PE 0? If the former we need to limit output to PE 0 only.

Reply:
This is called once per `Ck::IO::open` invocation if the write stripe size has not been specified. This is just a raw function call, so it should execute once on whatever PE makes the call (it kicks off some messaging internally, but that's not relevant to this patch).

Reply:
Would it make sense to run this detection only once per execution of the application?

Reply:
No, this detects the stripe settings for the particular file to be written; different files in different parts of the file system can have different stripe settings set, so this should ideally be dynamic.

Reply:
Ok, in that case, it might make sense to also print the filename of the file for which the detection fails.

Reply:
Good idea, I've just done that.